action.yaml

```diff
@@ -343,17 +343,8 @@ runs:
       env:
         GITHUB_EVENT_WORKFLOW_RUN_HEAD_SHA: ${{ github.event.workflow_run.head_sha }}
 
-    - name: Finalize check run
-      if: always() && env.INPUT_CHECK_RUN_ID
-      continue-on-error: true
-      shell: bash
-      run: |
-        "${TRUNK_PATH}" check finalize-github-check-run \
-          --target "${INPUT_TARGET_CHECKOUT}" \
-          --check_run_id "${INPUT_CHECK_RUN_ID}" \
-
```

Comment:
sanity checking - we don't need this anymore, because we're tracking the state of everything in the backend, right?

Reply:
correct